content/en/docs/tasks/administer-cluster/enabling-topology-aware-hints.md

```diff
@@ -0,0 +1,40 @@
+---
+reviewers:
+- robscott
+title: Enabling Topology Aware Hints
+content_type: task
+---
+
+<!-- overview -->
+{{< feature-state for_k8s_version="v1.21" state="alpha" >}}
+
+_Topology Aware Hints_ enable topology aware routing with topology hints
+included in EndpointSlices. This approach tries to keep traffic close to where
+it originated from. This can result in decreased costs and improved performance.
+
+## {{% heading "prerequisites" %}}
+
+  {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
+
+The following prerequisites are needed in order to enable topology aware hints:
+
+   * Kubernetes 1.21 or later
```

Comment:
The version check shortcode covers this for you if you set `min-kubernetes-server-version` in the front matter.

Reply:
Updated, thanks!